Should throw exception because initial instance is not in phase with start instance
printStackTrace
e
getSyncDatasetEvents
getMessage
${coord:current(0)}
out-of-phase-coordinator.xml
getCause
System
fail
contains
assertTrue
initial-instance is not in phase with start-instance
2009-08-20T18:00Z
testCoordEndOfMonthsInitialInstaceNotInPhase
${coord:endOfMonths(-1)}
